modules/virt-checking-storage-configuration.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-running-cluster-checkups.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-checking-storage-configuration_{context}"]
+= Running a storage checkup
+
+Use a predefined checkup to verify that the {product-title} cluster storage is configured optimally to run {VirtProductName} workloads.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `ServiceAccount`, `Role`, and `RoleBinding` manifest for the storage checkup:
++
+.Example service account, role, and rolebinding manifest file
+[%collapsible]
+====
+[source,yaml]
```

Comment:
Hmm, since the manifests are provided in the UI is it necessary to reproduce them here?  I worry about keeping this example in sync with any changes to the product.  Maybe it would be better to provide instructions for finding this yaml in the GUI?

Reply:
It's consistent with the above Network Latency and DPDK checkup doc, however I agree that documenting the UI-way to do it is necessary, as I guess that's the way we expect most customers to use the checkups.

Reply:
Based on the discussion on Slack, a separate Jira will be created to cover the UI procedure. This PR will cover the CLI procedure.